frontend/src/components/Buttons/TypeButton.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <v-menu>
+  <v-menu :disabled="disabled">
     <template #activator="{ props: menuProps }">
       <v-btn
-        v-if="!$vuetify.display.smAndDown && (model.length === 0 || model[0])"
```

Comment:
Why? You're cluttering the setup with this change imo. At the past, @ThibaultNocchi and me discussed about the different methods of importing icons and we agreed that having icons both defined as prop or as components is fine, since the templates can get very cluttered with how Vuetify uses icon as slots. However, in this case, we take even more space (when icons use around 1:1 of the space)

Reply:
I like the idea of the code part of the template explicitly declaring all the requirements that the template will need, rather than have "global" variables in the template. But if there's been a decision to not do this then I will revert it.

Reply:
@aweebs I thought this was the reasoning behind, so somehow we agreed beforehand :).

And yes, I agree some things like this are confusing in Vue (like the ref unwrapping). No strong feelings about it, just wanted to be sure if it was conscious and not because you thought it wasn't working, for instance.